riemann/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:8.1
```

Comment:
We'll need to build our own image even if we derive it from an existing base, because we still need to supply an appropriate `/etc/riemann/riemann.config`.